kubernetes/wait-until-kubectl-job-is-completed.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash -e
+
+job_name=$1
+timeout=$2
+
+time_counter=0
+until [[ "$(kubectl get job $job_name --output jsonpath='{.status.conditions[?(@.type=="Complete")].status}')" == "True" ]] ; do
+    if [[ "$timeout" == "$time_counter" ]]; then
+        echo "Timeout has occurred"
```

Comment:
Let's reword it to "Timeout: job `<job_name>` has not completed in `<timeout>`. Not waiting any longer."